include/deal.II/fe/fe_nedelec.h

```diff
@@ -139,6 +139,253 @@ DEAL_II_NAMESPACE_OPEN
  * The points needed are those of QGauss<sub>k+1</sub> on each edge and
  * QGauss<sub>k+2</sub> on each face and in the interior of the cell (or none
  * for N<sub>1</sub>).
+ *
+ * <h4>FE_Nedelec(0)</h4>
+ *
+ * For the lowest order N&eacute;d&eacute;lec element, we have a single shape
+ * function associated with each edge (i.e., the tangential component of each
+ * shape function is non-zero on only one edge).
+ *
+ * In 2D, these shape functions, normalized such that the maximum magnitude is
```

Comment:
Are the shape functions actually normalized, or just the plots?

Reply:
Only the plots were normalized. I have changed the visualization based on your comment below, so this portion no longer applies. However, if I were to retain the 3D perspective, I would clarify that the plots are normalized instead.